addons/im_livechat/static/src/core/common/chat_window_model_patch.js

```diff
@@ -13,30 +13,34 @@ patch(ChatWindow.prototype, {
         this.livechatStep = CW_LIVECHAT_STEP.NONE;
     },
 
-    async close(...args) {
+    async close(options = {}) {
         if (this.thread?.channel_type !== "livechat") {
-            return super.close(...args);
+            return super.close(...arguments);
+        }
+        if (options.force) {
+            this.livechatStep = CW_LIVECHAT_STEP.NONE;
+            super.close(...arguments);
         }
```

Comment:
In my PR this didn't look necessary, so no need to pass `force: true`. Do you have a case for it being needed?

Reply:
When someone deletes a livechat from channels list and if `force` is not provided then it will not unpin and remove the chatwindow from both client and internal user.

Reply:
As it would be handled by `channel/delete` notification. And there won't be any further notification to handle the `CLOSE` step.

Reply:
Couldn't add a test case because a traceback occurs when deleting a live chat while the messaging menu is loaded. Created a task for the issue.